app/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -1,261 +1,172 @@
-import time
-
 import pytest
 from nimbus.pages.home import HomePage
 from nimbus.pages.summary import SummaryPage
-from selenium.common.exceptions import NoSuchElementException, TimeoutException
+from nimbus.remote_settings.pages.dashboard import Dashboard
+from nimbus.remote_settings.pages.login import Login
 
 
 @pytest.mark.run_per_app
 def test_create_new_experiment_approve_remote_settings(
     selenium,
     base_url,
+    kinto_url,
+    kinto_review_url,
+    experiment_url,
     default_data,
     create_experiment,
-    perform_kinto_action,
-    timeout_length,
 ):
     selenium.get(base_url)
     home = HomePage(selenium, base_url).wait_for_page_to_load()
-    current_experiments = None
-    try:
-        current_experiments = len(home.tables[0].experiments)
-    except TimeoutException:
-        current_experiments = 0
 
-    review = create_experiment(selenium, home, default_data)
-    review.launch_without_preview.click()
-    review.request_review.click_launch_checkboxes()
-    review.request_review.request_launch_button.click()
-    review.approve(timeout_length)
+    summary = create_experiment(selenium, home, default_data)
+    summary.launch_without_preview.click()
+    summary.request_review.click_launch_checkboxes()
+    summary.request_review.request_launch_button.click()
+    summary.approve()
 
-    perform_kinto_action(selenium, base_url, "approve")
+    selenium.get(kinto_url)
+    kinto_login = Login(selenium, kinto_url).wait_for_page_to_load()
+    kinto_login.login()
 
-    selenium.get(base_url)
-    # refresh until the experiment shows up
-    for attempt in range(timeout_length):
-        try:
-            home = HomePage(selenium, base_url).wait_for_page_to_load()
-            new_experiments = len(home.tables[0].experiments)
-            assert new_experiments > current_experiments
-        except AssertionError:
-            time.sleep(2)
-            selenium.refresh()
-        else:
-            break
-    # Check it's live
-    home = HomePage(selenium, base_url).wait_for_page_to_load()
-    live_experiments = home.tables[0]
-    assert "Live" in home.active_tab_text
-    for item in live_experiments.experiments:
-        if default_data.public_name in item.text:
-            item.click()
-            break
-    summary_page = SummaryPage(selenium, base_url).wait_for_page_to_load()
-    assert "live" in summary_page.experiment_status.lower()
+    selenium.get(kinto_review_url)
+    kinto_dashboard = Dashboard(selenium, kinto_review_url).wait_for_page_to_load()
+    kinto_dashboard.approve()
```

Comment:
Did you try using ```.open()```?

Reply:
I did but something else had to change so I left out and was gonna do it as a followup.